src/app/pages/setting/setting.component.html

```diff
@@ -0,0 +1,340 @@
+<div class="header bg-gradient-danger pb-8 pt-5 pt-md-8">
+    <div class="container-fluid">
+      <div class="header-body">
+        <!-- Card stats -->
+        <h1 class="display-2 text-white">Hello</h1>
+        <p class="text-white mt-0 mb-5">This is your profile setting page.</p>
+      </div>
+    </div>
+  </div>         
+        <form>
+         <div ng-show="receiveemail" class="form-check">
+              <input type="checkbox" class="form-check-input" id="Check1">
+              <label class="form-check-label" for="Check1">Receive email</label> <br />
+            </div> <br />
+            <div ng-show="receivetext" class="form-check">
+                <input type="checkbox" class="form-check-input" id="check2">
+                <label class="form-check-label" for="Check2">Receive text</label>
+            </div> <br />
+         </form>     
+
+            <form action="//www.html.am/html-codes/textboxes/submitted.cfm">
+                <textarea name="myTextBox" cols="50" rows="5">
+                 How many emails per day...
+                </textarea>
+                <br />
+                <input type="Submit" class="btn-sm btn btn-success" />
+                 <br />
+                 <br />
+
+               <form action="//www.html.am/html-codes/textboxes/submitted.cfm">
+                    <textarea name="myTextBox" cols="50" rows="5">
+                     How many emails per week...
+                    </textarea>
+                    <br />
+                    <input type="Submit" class="btn-sm btn btn-success" />
```

Comment:
There is no need to have mutliple submit buttons here and there one submit button is good.